docs/language-basics/documentation.md

```diff
@@ -3,12 +3,8 @@ id: documentation
 title: Documentation
 ---
 
-# Documentation
-
 Documentation is a vital aspect of any API. TypeSpec offers several ways to document your API, including doc comments and decorators.
 
-# Approaches to documenting APIs
-
```

Comment:
Hmm, should this be an h2 (`##`) and everything under it have their size decreased by 1?